ci-operator/step-registry/mirror-images/by-oc-adm/mirror-images-by-oc-adm-commands.sh

```diff
@@ -78,11 +77,11 @@ registry_cred=$(head -n 1 "/var/run/vault/mirror-registry/registry_creds" | base
 jq --argjson a "{\"${MIRROR_REGISTRY_HOST}\": {\"auth\": \"$registry_cred\"}}" '.auths |= . + $a' "${CLUSTER_PROFILE_DIR}/pull-secret" > "${new_pull_secret}"
 oc registry login --to "${new_pull_secret}"
 
-mirror_crd_type='icsp'
-regex_keyword_1="imageContentSources"
-if [[ "${ENABLE_IDMS}" == "yes" ]]; then
-    mirror_crd_type='idms'
-    regex_keyword_1="imageDigestSources"
+mirror_crd_type='idms'
```

Comment:
We still have many old versions of ci jobs, especially for chain upgrade ci jobs from old OCP versions, we can not guarantee they always work with `idms`, suggest to untouched the default value, but set `ENABLE_IDMS` in the install workflow or ci jobs to overwrite the default behavior.